tests/libs/test_pydantic.py

```diff
@@ -724,3 +724,13 @@ class MyParent(Parent):
 
     assert "data_dictionary" not in schema
     assert "data_list" not in schema
+
+
+def test_parent_nullable_means_children_nullable():
+    class MyParent(BaseModel):
```

Comment:
@joscha could you add the reverse test?  Another field with a non optional child model and test that this is not nullable? Then make sure the conflicts are solved and the linter passes, then we can merge this :)

Reply:
ah yes, good point, will do!

Reply:
inverse test in 74af54c